x-pack/platform/plugins/shared/fleet/server/services/output.ts

```diff
@@ -128,7 +128,7 @@ function outputSavedObjectToOutput(so: SavedObject<OutputSOAttributes>): Output
   return {
     id: outputId ?? so.id,
     ...atributes,
-    ...(ssl ? { ssl: JSON.parse(ssl as string) } : {}),
+    ...(typeof ssl === 'string' ? { ssl: JSON.parse(ssl as string) } : {}),
```

Comment:
ssl can still be an invalid JSON, even though it's a string. Shouldn't we wrap `JSON.parse` in try catch?

Reply:
good point, updated code and added unit tests